Created Output Log File: /workdir/tasks_run/data/nfb_logs/p004_calculator_script_20240911_23h46m42s.txtRunning Main Calculation Script ... 
Starting Block1 ... 
========================================
Starting Block1, Trial 1... 
========================================
Waiting For New File ...
New File Found In Dir...
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/36288010028664961085.dcm
Error:
name 'file_handler' is not defined
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 26, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 23, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["mean_activation"] = calculations_MW.get_mean_activation(roi_mask=dictionary["whole_session_data"]["roi_mask_path"], nifti_image_path=dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"])
  File "/workdir/tasks_run/scripts/calculations_MW.py", line 39, in get_mean_activation
    log.print_and_log(f"Mean ROI Activation: {mean_activation}")
  File "/workdir/tasks_run/scripts/log.py", line 70, in print_and_log
    log_name = file_handler.get_most_recent(action="txt_output_log")
NameError: name 'file_handler' is not defined

Retries left: 3
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/36288010028664961085.dcm
Error:
name 'file_handler' is not defined
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 26, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 23, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["mean_activation"] = calculations_MW.get_mean_activation(roi_mask=dictionary["whole_session_data"]["roi_mask_path"], nifti_image_path=dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"])
  File "/workdir/tasks_run/scripts/calculations_MW.py", line 39, in get_mean_activation
    log.print_and_log(f"Mean ROI Activation: {mean_activation}")
  File "/workdir/tasks_run/scripts/log.py", line 70, in print_and_log
    log_name = file_handler.get_most_recent(action="txt_output_log")
NameError: name 'file_handler' is not defined

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Retries left: 2
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/88071641823349728863.dcm
Error:
name 'file_handler' is not defined
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 26, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 23, in run_trial
    dictionary[f"block{block}"][f"trial{trial}"]["mean_activation"] = calculations_MW.get_mean_activation(roi_mask=dictionary["whole_session_data"]["roi_mask_path"], nifti_image_path=dictionary[f"block{block}"][f"trial{trial}"]["nifti_path"])
  File "/workdir/tasks_run/scripts/calculations_MW.py", line 39, in get_mean_activation
    log.print_and_log(f"Mean ROI Activation: {mean_activation}")
  File "/workdir/tasks_run/scripts/log.py", line 70, in print_and_log
    log_name = file_handler.get_most_recent(action="txt_output_log")
NameError: name 'file_handler' is not defined

Getting New Dicom Instead...
Waiting For New File ...
---- Keyboard Interrupt Detected ----
What Would You Like to Do?
(1) Continue With The Block
(2) End The Session
(3) Start New Block
Ok, Let's End the Session...
Nifti Outdir Cleared
Ending Session Due to: keyboard interrupt
dicom dir using: /workdir/tasks_run/data/sambashare/test_dir
